index.bs

```diff
@@ -4422,6 +4483,203 @@ Of the extended attributes defined in this specification, only the [{{Exposed}}]
+
+<div algorithm>
+A module has a [=list=] of
+<dfn id="dfn-module-member" export lt="module member" for=module>module members</dfn> which are
+[=interfaces=], [=read only=] [=regular attributes=], and [=regular operations=].
+
+The [=module/module members=] of a [=module=] |module| are given by the following steps:
+
+    1.  Let |result| be « ».
+    1.  Let |specifier| be |module|'s [=module/specifier=].
+    1.  For each [=module declaration=] |declaration| whose [=module declaration/specifier=] is
+        |specifier|:
+        1.  For every member |member| in |declaration|'s [=module declaration/module
+            declaration members|members=]:
+            1.  [=list/Append=] |member| to |result|.
+    1.  Let |partials| be « ».
+    1.  For each [=partial module declaration=] |declaration| whose [=partial module
+        declaration/specifier=] is |specifier|:
+        1.  Let |partialMembers| be |declaration|'s  [=partial module declaration/partial module
+            declaration members|members=].
+        1.  Assert: |partialMembers| is not [=list/is empty|empty=].
+        1.  [=list/Append=] |partialMembers| to |partials|.
+    1.  Sort |partials| lexicographically by the [=identifier=] of each [=list/item=]'s first
+        [=list/item=].
+    1.  [=list/iterate|For each=] |partialMembers| of |partials|:
+        1.  [=list/iterate|For each=] |member| of |partialMembers|:
+            1.  [=list/Append=] |member| to |result|.
+    1.  Return |result|.
+
+Note: The order that members appear in this list has significance for property enumeration in the
+<a href="#es-modules">ECMAScript binding</a>.
+
+Issue: Exclude [=partial interfaces=]?
+</div>
+
+For all [=modules=] |module|, all [=interfaces=] and [=partial interfaces=] that are in |module|'s
+[=module/module members=] have their [=interface/enclosing module=] set to |module|.
+
+<div algorithm>
+  The <dfn for=module>url</dfn> of a [=module=] |module| is given by the following steps:
+
+    1.  Let |specifier| be |module|'s [=module/specifier=].
+    1.  Return the [=concatenation=] of « "<code>import:std:</code>", |specifier| ».
+</div>
+
+Note that like [=namespaces=], modules do not create types.
+
+Rather than defining behavior when accessed, the [=attributes=] of a [=module=] act as a simple
+storage location. The relevant language binding defines the steps to "set a module attribute" to
+an IDL value of the type of the [=attribute=].
+
+Note: See the ECMAScript binding's definition of [=set a module attribute=].
+
+Each module may have <dfn for=module>evaluation steps</dfn> which are executed when the module is
+first imported.
+These steps have access to a <dfn>this module</dfn> value, which is the [=module=] being
+evaluated, and a [=Realm=].
+They may throw an [=exception=], which prevents the module from being accessed.
+The [=module/evaluation steps=] for a module |m| must invoke the "set a module attribute"
```

Comment:
I'm not a big fan of this "quotes surrounding a term" in place of cross-links, but I understand that the alternative is defining a language-agnostic term and a ES-specific term which is also annoying. I'm not sure what to do here.

Reply:
Let's leave it for now.